src/pages/overview/pdf-embed-api/modernviewer.md

```diff
@@ -142,9 +144,38 @@ All APIs and callbacks will work as expected with significant improvement in use
 </html>
 ```
 
-Developers using the [main.js](https://documentservices.adobe.com/view-sdk/main.js) script in their websites 
-will continue to see the legacy PDF viewer and all PDF Embed API functionality 
+### Legacy PDF viewer integration
+
+```html
```

Comment:
I do not see the point of including the script sample here. It should be removed.